detekt-config.yml

```diff
@@ -213,7 +213,7 @@ naming:
     minimumVariableNameLength: 1
   VariableNaming:
     active: true
-    variablePattern: '[a-z][A-Za-z0-9]*'
+    variablePattern: '(_)?[a-z][A-Za-z0-9]*'
```

Comment:
🤔  We don't use `_` as the prefix for a non-private variable.

Reply:
correct, fixed

Reply:
@hoangnguyen92dn we need this for protected fields https://github.com/nimblehq/android-templates/pull/62/files#diff-0cbbe6c8fd6798d5dc14330d3a7012d4a92a30727ddfb72849c96da3b291a982R19-R20 🤔

Reply:
🆗